juniper_official/Solutions/Pfe-wedge-detection/lueachip-pattern-set.rule

```diff
@@ -0,0 +1,69 @@
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "If single log, no action.  If numerous errors, reset the MPC and monitor for recurrence.";
+                }
+                key-fields fpc_number;
+            }
+            pattern LUEA_EVENT002 {
+                event-id LUEA_EVENT002;
+                filter "fpc%{NUMBER:fpc_number}.*lmem data error";
+                description "LMEM data errors are parity errors and the software will repair those errors automatically. If the number of errors exceed the threshold of 7, then a particular zone will be disabled for further usage. If this threshold is reached for shared zones, the entire PPE (Packet Processing Engine) will become disabled";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "Reset FPC only if Wedge is seen.  Otherwise no action";
+                }
+                key-fields fpc_number;
+            }
+            pattern LUEA_EVENT003 {
+                event-id LUEA_EVENT003;
+                filter "fpc%{NUMBER:fpc_number}.*Errors async xtxn error";
+                description "The \"Errors sync axtxn error\" message is likely caused by a software related error condition accessing illegal memory space.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "Look for other accompanying errors";
+                }
+                key-fields fpc_number;  
+            }
+            pattern LUEA_EVENT004 {
+                event-id LUEA_EVENT004;
+                filter "fpc%{NUMBER:fpc_number}.*Errors sync xtxn error";
+                description "The \"Errors sync xtxn error\" message is likely to be caused due to Hardware related error condition since synchronous transaction to external device is not possible.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
+                    value "Look for other accompanying errors";
+                }
+                key-fields fpc_number;
+            }
+            pattern LUEA_EVENT005 {
+                event-id LUEA_EVENT005;
+                filter "fpc%{NUMBER:fpc_number}.*Uninitialized Read Error";
+                description "The \"Uninitialized Read Error\" message is caused by a transient hardware or software error.Access to unintialized memory with predefined patterns are software errors and the rest are hardware errors.";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
```

Comment:
And here..

Reply:
@ConstrainedRandomVar , added trigger based on count